skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h

```diff
@@ -43,9 +45,44 @@ struct BaseActorValues
   }
 };
 
-inline BaseActorValues GetBaseActorValues(uint32_t baseId,
+inline BaseActorValues GetBaseActorValues(espm::Loader& espm, uint32_t baseId,
                                           uint32_t raceIdOverride)
 {
   BaseActorValues baseActorValues;
+
+  auto form = espm.GetBrowser().LookupById(baseId);
+
+  if (form.rec->GetType() == "NPC_") {
+    auto npc = espm::Convert<espm::NPC_>(form.rec);
+
+    espm::CompressedFieldsCache compressedFieldsCache;
+    auto raceId = npc->GetData(compressedFieldsCache).race;
+    raceId = raceId ? raceId : 0x00013746;
+    if (raceIdOverride != 0) {
```

Comment:
```suggestion
    if (raceIdOverride != 0) {
```
Is that it?

Reply:
@d0gge raceIdOverride must override raceId. here raceId overrides raceIdOverride . consider unit testing this so we could simpler discuss the logic